source/reference/ejson.txt

```diff
@@ -0,0 +1,55 @@
+.. _mongosh-ejson:
+
+=====
+EJSON
+=====
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+..
+   COMMENT: These URLs aren't reused, they're just for editing clarity
+
+.. |bsonUrl| replace:: `BSON <https://www.mongodb.com/basics/bson>`__
+
+.. |ejsonUrl| replace:: `EJSON <https://www.npmjs.com/package/bson#EJSON>`__
+
+.. |bsonParseUrl| replace:: `Node.js BSON parser <https://www.npmjs.com/package/bson>`__
+
+MongoDB uses |bsonUrl|, a binary serialization format, to store
+documents and to exchange data. ``mongosh`` exposes the |ejsonUrl|
+interface from the |bsonParseUrl| to help you transform your data when
+you work with ``mongosh``. Use the ``EJSON`` interface when you need to
+transform data. 
```

Comment:
This text should somewhere expand `EJSON` as `Extended JSON` to clarify what the acronym actually stands for, and maybe also state that EJSON is a JSON-compatible way of _representing_ BSON values.